modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1199,6 +1199,9 @@
      */
     public static final String IGNITE_TX_OWNER_DUMP_REQUESTS_ALLOWED = "IGNITE_TX_OWNER_DUMP_REQUESTS_ALLOWED";
 
+    /** Maximum count of active transactions per connection for thin clients. */
+    public static final String IGNITE_THIN_MAX_ACTIVE_TX_PER_CONNECTION = "IGNITE_THIN_MAX_ACTIVE_TX_PER_CONNECTION";
```

Comment:
I think this should go to `ClientConnectorConfiguration` instead where we already have things like `maxOpenCursorsPerConn`.

Reply:
I was thinking about that, but `maxActiveTxPerConnection` is only reliable to thin clients, while this `ClientConnectorConfiguration` is used to configure JDBC and ODBC connections as well. Property `maxOpenCursorsPerConn` is reliable to all connection types, but property `maxActiveTxPerConnection` is useless for JDBC and ODBC and may be confusing.

Reply:
I think we better should discuss it on dev-list. I will start a conversation.